src/Okta.Sdk.IntegrationTests/ApplicationScenarios.cs

```diff
@@ -372,11 +372,12 @@ public async Task AddOpenIdConnectApp()
         {
             var client = TestClient.Create();
             var guid = Guid.NewGuid();
+            var testClientId = $"{nameof(AddOpenIdConnectApp)}_TestClientId";
 
             var createdApp = await client.Applications.CreateApplicationAsync(new CreateOpenIdConnectApplication
             {
                 Label = $"dotnet-sdk: AddOpenIdConnectApp {guid}",
-                ClientId = "0oae8mnt9tZcGcMXG0h3",
+                ClientId = testClientId,
```

Comment:
Why this change is needed?

Reply:
This was causing integration tests to fail because the same value was being used by multiple tests.  There was a 400 returned by the server because an object with the same clientId already exist.

Reply:
I could move this change into a separate branch/set of changes so usertypes is isolated if you prefer.

Reply:
oh right! this is the one you mentioned today. It's Ok, we can keep it here.